models/staging/stg_ga4__events.sql

```diff
@@ -1,6 +1,8 @@
 -- This staging model contains key creation and window functions. Keeping window functions outside of the base incremental model ensures that the incremental updates don't artificially limit the window partition sizes (ex: if a session spans 2 days, but only 1 day is in the incremental update)
 with base_events as (
     select * from {{ ref('base_ga4__events')}}
+    where true
+    {{ filter_date_range(start_date, end_date, event_date_dt) }}
```

Comment:
There are more staging models we use, if this the only on where we need to add this line? 
I'm not sure we even need this line as we can filter down the line. It might also interfere with the window functions in this code. We need to ask do we want to filter in a place where we rarely look? In 3 months, do we still know whats happening here? 
Is this a view or a table?